pkg/controller/forwarding/forwarding_controller.go

```diff
@@ -98,9 +98,12 @@ func (r *ReconcileForwarding) Reconcile(request reconcile.Request) (reconcile.Re
 	//check for instancename and then update status
 	var reconcileErr error = nil
 	if instance.Name != singletonName {
-		instance.Status = logforwarding.NewForwardingStatus(logforwarding.LogForwardingStateRejected, logforwarding.LogForwardingReasonName, singletonMessage)
+		instance.Status = logforwarding.NewForwardingStatus()
```

Comment:
can we revert this? it makes more sense to me that we would return a status based on something we return and not have to set the fields again after.

Reply:
reverted to be more like a proper constructor